packages/modelserver-node/src/client/model-server-client.ts

```diff
@@ -383,6 +398,14 @@ class DefaultTransactionContext implements TransactionContext {
     }
 
```

Comment:
I like the doc on performTriggers, can we add a short line to all other public methods? (I know it is a bit out of context on this pr)